packages/core/src/builtin/initCore.ts

```
@@ -0,0 +1,15 @@
+import { notetypeRegistry } from "../notes";
+import { builtinNotetypes } from "builtin/notetypes";
+
+export interface InitCoreOptions {
+  // プラグインなど予定
+}
+
+export async function initCore(options: InitCoreOptions) {
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)